examples/sample_costs.csv

```diff
@@ -0,0 +1,267 @@
+CountryName,RegionName,C1_School closing,C2_Workplace closing,C3_Cancel public events,C4_Restrictions on gatherings,C5_Close public transport,C6_Stay at home requirements,C7_Restrictions on internal movement,C8_International travel controls,H1_Public information campaigns,H2_Testing policy,H3_Contact tracing,H6_Facial Coverings
+Afghanistan,,1,1,1,1,1,1,1,1,1,1,1,1
+Albania,,1,1,1,1,1,1,1,1,1,1,1,1
+Algeria,,1,1,1,1,1,1,1,1,1,1,1,1
+Andorra,,1,1,1,1,1,1,1,1,1,1,1,1
+Angola,,1,1,1,1,1,1,1,1,1,1,1,1
+Argentina,,1,1,1,1,1,1,1,1,1,1,1,1
+Aruba,,1,1,1,1,1,1,1,1,1,1,1,1
+Australia,,1,1,1,1,1,1,1,1,1,1,1,1
+Austria,,1,1,1,1,1,1,1,1,1,1,1,1
+Azerbaijan,,1,1,1,1,1,1,1,1,1,1,1,1
+Bahamas,,1,1,1,1,1,1,1,1,1,1,1,1
+Bahrain,,1,1,1,1,1,1,1,1,1,1,1,1
+Bangladesh,,1,1,1,1,1,1,1,1,1,1,1,1
+Barbados,,1,1,1,1,1,1,1,1,1,1,1,1
+Belarus,,1,1,1,1,1,1,1,1,1,1,1,1
+Belgium,,1,1,1,1,1,1,1,1,1,1,1,1
+Belize,,1,1,1,1,1,1,1,1,1,1,1,1
+Benin,,1,1,1,1,1,1,1,1,1,1,1,1
+Bermuda,,1,1,1,1,1,1,1,1,1,1,1,1
+Bhutan,,1,1,1,1,1,1,1,1,1,1,1,1
+Bolivia,,1,1,1,1,1,1,1,1,1,1,1,1
+Bosnia and Herzegovina,,1,1,1,1,1,1,1,1,1,1,1,1
+Botswana,,1,1,1,1,1,1,1,1,1,1,1,1
+Brazil,,1,1,1,1,1,1,1,1,1,1,1,1
+Brazil,Acre,1,1,1,1,1,1,1,1,1,1,1,1
```

Comment:
Brazil regions should not be there. This is probably coming from `covid_xprize/validation/data/fixed_equal_costs.csv`, for which I created PR https://github.com/leaf-ai/covid-xprize/pull/181